app/_hub/kong-inc/injection-protection/overview/_index.md

```diff
@@ -81,6 +81,12 @@ Logs are automatically collected when you enable the Injection Protection plugin
     * [UDP Log](/hub/kong-inc/udp-log/)
 * [{{site.konnect_short_name}} Advanced Analytics](/konnect/analytics/) <!--though i'm not sure if they will do anything with them yet--> 
 
+The following is an example of the information that is logged by the Injection Protection plugin:
+
+```
+threat detected: 'sql', action taken: log_only, found in path_and_query, query param value: foo: insert into test
+```
+
```

Comment:
```suggestion
Where:
* The plugin detected a SQL injection threat
* It created a log entry but took no other actions
* The threat was found in the request path or query
* The inserted snippet is `foo: insert into test`
```
I'd add something like this to give the log message a tiny bit of context. Not sure if I got it all correct, so please double-check.